src/app/policies/views/PolicyListView.vue

```diff
@@ -56,7 +56,7 @@
                           <p>
                             <strong>Warning</strong> This policy is experimental. If you encountered any problem please open an
                             <a
-                              href="https://github.com/kumahq/kuma/issues/new/choose"
+                              :href="env('KUMA_FEEDBACK_URL')"
```

Comment:
Could you change this to use `t`. We are moving away from using `env()` for these types of things now that we have a proper i18n layer.

Reply:
Happy to do this in a follow up, but I don’t have the time for this currently.

Reply:
How long is this going to take you? Longer than 2 minutes?